skills/bmad-module-builder/scripts/tests/test-validate-module.py

```
@@ -162,6 +163,73 @@ def test_empty_csv():
+def test_short_row_does_not_crash():
+    """A CSV row with fewer fields than the header must not crash the validator and
+    must be reported as a column-count mismatch. DictReader fills the missing
+    columns with None by default, so the validator's `.strip()` calls would raise
+    AttributeError on a short row — restval="" keeps them safe. Regression test."""
+    with tempfile.TemporaryDirectory() as tmp:
+        tmp = Path(tmp)
+        # Only 5 of the 13 columns present (the remaining 8 are missing entirely).
+        csv_rows = 'Test Module,tst-foo,Do Foo,DF,Does foo\n'
+        module_dir = create_module(tmp, skills=["tst-foo"], csv_rows=csv_rows)
+
+        code, data = run_validate(module_dir)
```

Ruff (0.15.13)

[warning] Unpacked variable `code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)